Assets/Scripts/TapeInteract.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.InputSystem;
+
+/// <summary>
+/// This script should be added on the PlayerControler Object.
+/// Usess raycasts to see whats in front of the player, to be interacted with.
+/// Shows tooltip if Player can interact with object
+/// 
+/// Requires: The tape object tag to be set to "Tape"
+/// </summary>
+
+
+public class TapeInteract : MonoBehaviour {
+    [Tooltip("Add the playerObject here")]
+    [SerializeField]
+    GameObject PlayerObject;
```

Comment:
- Follow the code convention ( 1)All methods, public fields, classes, enums and types are PascalCase, 2) All method parameters, local variables and non-public fields are camelCase.)
- Put `= default` to every [SerializeField] so that we don't get warning in the Console.

```suggestion
    GameObject playerObject = default;
```